src/main/resources/wa-task-configuration-ia-asylum.dmn

```diff
@@ -202,7 +202,7 @@
                        "reviewAppealSetAsideUnderRule32",
                        "reviewRemittedAppeal", "processFeeRefund",
                        "reviewDraftAppeal",
-                       "printAndSendHoBundle"
+                       "printAndSendHoBundle", "printAndSendHearingRequirements"
```

Comment:
For easier to rebase in the future, can you add this on the next line? Follow the format above